pybikes/data/stadtrad_hamburg.json

```diff
@@ -0,0 +1,15 @@
+{
+  "instances": [
+    {
+      "tag": "stadtrad_hamburg",
```

Comment:
This tag shouldn't pass the tests. My bad on sloppy triggering the tests on travis. Seems it failed but exit code was all good... 
https://travis-ci.org/eskerda/pybikes/builds/446204731#L512

Reply:
I fixed the tag name in commit f369942. Now the tests on Travis report no failures anymore.